_posts/2023-07-19-style-workflow.md

```diff
@@ -0,0 +1,82 @@
+ 
+The new style workflow aligns our approach to quality assurance with our approach to development, and you can choose which rules to turn on and whether to run the tool in the command line or integrate it with your IDE. Our intent is for this new mechanism to save us time during release cycles while still allowing us to achieve our high quality bar.
+
+## Installing Vale
+
+To install Vale, run `brew install vale`. 
+
+You can use the tool locally in two ways:
+
+1. **From the command line**. Run Vale on the whole directory (`vale .`) or a particular file (`vale /path/to/file`). Vale reports errors in the command line, as shown in the following image.
+
+    ![Command-line error example](/assets/media/blog-images/2023-07-19-style-workflow/command-line-error.png){: .img-fluid }    
+
+1. **As an integrated IDE extension**. Vale integrates with many IDEs, including VS Code, IntelliJ IDEA and other JetBrains IDEs, Sublime, and Emacs. To integrate Vale in VS Code, follow these steps:
+
+    * In the left panel, select **Extensions** and search for **Vale VSCode**. 
+    * Select **Install** and restart VS Code. 
+
+    The Vale extension highlights problems with squiggly underlines, similarly to Microsoft Word. You can hover over the underlined text to see the problem and optionally view the associated rule, which in most cases provides a link to the relevant section of the project style guide. For some problems, you can also choose **Quick Fix**, which fixes the problem automatically, as shown in the following image.
+
+    ![Vale error example](/assets/media/blog-images/2023-07-19-style-workflow/vale-error.png){: .img-fluid }
+
+    NOTE: The Vale extension only refreshes when you save the file.
+
+## Reporting levels
+ 
+There are **three levels of reporting** in the style workflow, and you can choose which levels are displayed:
+
+* **Error**: Incorrect usage.
+* **Warning**: Usage that is grammatically correct but does not adhere to the style guide.
+* **Suggestion**: Stylistic choices that do not follow best practices. Currently, only passive voice and future tense are flagged as suggestions.
+
+By default, Vale is set up to report *errors* and *warnings*. **To select the level of reporting**, follow these steps:
+
+* In the left panel, select **Extensions** and then select **Vale VSCode**.
+* Select the settings (gear) icon in the lower-right corner of the Vale VSCode extension and select **Extension settings**.
+* In the Vale > Vale CLI: **Min Alert Level** dropdown list, select **suggestion** or any other level.
+
+## Helpful hints
+
+* Code blocks that specify a language after the three tic marks are skipped, for example:
+    ````
+    ```json
+    GET _cat/aliases?v
+    ```
+    ````
+
```

Comment:
Is there an error causing the text after this point to maintain the preceding formatting?

Reply:
Not that I see. This is all under the first bullet point so it's indented. Other than that, I think the formatting is fine.